src/main/java/edu/cornell/kfs/krad/service/impl/CuAttachmentServiceImpl.java

```diff
@@ -85,6 +85,8 @@ public Attachment createAttachment(PersistableBusinessObject parent, String uplo
             InputStream fileContentsToScan = new ByteArrayInputStream(fileContentsAsByteArray);
 
             if (!ScanResult.Status.PASSED.equals(antiVirusService.scan(fileContentsToScan).getStatus())) {
```

Comment:
Should any of the data from the returned ScanResult object be logged as well?

Reply:
good point, I shall add scan results details